src/storybook/components/unstyled-list-item/unstyled-list-item.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../../styles/global-css-settings.scss";
+@import "../../styles/typography.scss";
+
+.monday-storybook-unstyled-list-item {
+  font-size: 1rem;
```

Comment:
all our fonts in the storybook are defined by using fixed fonts units (px), doesn't it will be problematic that only the list define with rem units?